sdk/src/salt.rs

```diff
@@ -55,20 +55,11 @@ impl Default for DefaultSalt {
 
 impl SaltGenerator for DefaultSalt {
     fn generate_salt(&self) -> Option<Vec<u8>> {
-        #[cfg(feature = "openssl_sign")]
+        #[cfg(target_arch = "wasm32")]
         {
-            let mut salt = vec![0u8; self.salt_len];
```

Comment:
@mauricefisher64 check this one especially. Is there something special about `openssl`'s implementation of `rand_bytes` that we'd want to preserve? LMK and I'll adapt if so.

Reply:
The only requirement is it should be a cryptographically random number generator.